voicebox/connection.py

```diff
@@ -69,10 +92,18 @@ def receive_data(self):
             except ConnectionResetError:
                 logging.error("Connection to client lost: Adviced to kill connection")
             except OSError:
-
                 logging.info("Channel already closed: Adviced to kill connection")
 
     def send_message(self, message, msg_type=1):
```

Comment:
In the `send_message` method of `Connection` class, it would be better to return a status message or boolean value indicating the success or failure of the operation. Currently, the method doesn't return anything in case of success which might lead to ambiguity in understanding the method's outcome. [medium]